specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/SyncMemberGet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "syncgroupcrud-65440",
+    "serverName": "syncgroupcrud-8475",
+    "databaseName": "syncgroupcrud-4328",
+    "syncGroupName": "syncgroupcrud-3187",
+    "syncMemberName": "syncgroupcrud-4879",
+    "api-version": "2015-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "databaseType": "AzureSqlDatabase",
+          "serverName": "syncgroupcrud-3379.database.windows.net",
+          "databaseName": "syncgroupcrud-7421",
+          "userName": "myUser",
+          "syncDirection": "Bidirectional",
+          "syncState": "UnProvisioned"
```

Comment:
Examples doesn't show newly introduced properties, would be good to refer them here.

Reply:
Added usePrivateLinkConnection to examples.